src/Features/Blockcore.Features.Miner/BlockDefinition.cs

```diff
@@ -410,7 +410,9 @@ protected virtual void AddTransactions(out int nPackagesSelected, out int nDesce
                     packageSigOpsCost = modit.SigOpCostWithAncestors;
                 }
 
-                if (packageFees < this.BlockMinFeeRate.GetFee((int)packageSize))
```

Comment:
if you set BlockMinFeeRate to zero doe it not have the same effect?

Reply:
Yes, however, If the OP_RETURN data exists, we still want a fee, so if you look at the x42 override, I have the check for it there..

Reply:
Then just make a rule for x42 (and a mempool rule) and it wills till work no?

Reply:
I don't see how that would work, as other chains need the default behavior.